File: frontend/pyside6/api/store_api.py
import requests


BASE_URL = "http://127.0.0.1:8000"



def get_stores():

    response = requests.get(
        f"{BASE_URL}/stores"
    )

    response.raise_for_status()

    return response.json()



def create_store(name, location):

    response = requests.post(
        f"{BASE_URL}/stores",
        json={
            "name": name,
            "location": location
        }
    )

    response.raise_for_status()

    return response.json()

File: frontend/react/src/api/StoreApi.ts
import axiosInstance from './axiosInstance';
import type { StoreCreate, StoreResponse } from '../types/store';

// 매장 전체 조회
export const getStores = async (): Promise<StoreResponse[]> => {
  const response = await axiosInstance.get<StoreResponse[]>('/stores');
  return response.data;
};

// 매장 생성
export const createStore = async (data: StoreCreate): Promise<StoreResponse> => {
  const response = await axiosInstance.post<StoreResponse>('/stores', data);
  return response.data;
};